[🐛 Bug]: Broken types in `@wdio/global/types`

### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

8.2.0

### Node.js Version

latest

### Mode

Standalone Mode

### Which capabilities are you using?

_No response_

### What happened?

I am getting an error trying to build my setup with typescript

```
node_modules/@wdio/globals/types.d.ts:1:42 - error TS2694: Namespace '"/Users/kyrylo/project/node_modules/webdriverio/build/index"' has no exported member 'Browser'.

1 type BrowserType = import('webdriverio').Browser
```

### What is your expected behavior?

_No response_

### How to reproduce the bug.

Simply typescript project with wdio

### Relevant log output

```typescript
-
```


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

@christian-bromann These are the errors I see

```
node_modules/@wdio/globals/types.d.ts:1:42 - error TS2694: Namespace '"/Users/kyrylo/Projects/webdriverio-types-issue-example/node_modules/webdriverio/build/index"' has no exported member 'Browser'.

1 type BrowserType = import('webdriverio').Browser
                                           ~~~~~~~

node_modules/@wdio/globals/types.d.ts:2:42 - error TS2694: Namespace '"/Users/kyrylo/Projects/webdriverio-types-issue-example/node_modules/webdriverio/build/index"' has no exported member 'Element'.

2 type ElementType = import('webdriverio').Element
                                           ~~~~~~~

node_modules/@wdio/globals/types.d.ts:4:53 - error TS2694: Namespace '"/Users/kyrylo/Projects/webdriverio-types-issue-example/node_modules/webdriverio/build/index"' has no exported member 'MultiRemoteBrowser'.

4 type MultiRemoteBrowserType = import('webdriverio').MultiRemoteBrowser
                                                      ~~~~~~~~~~~~~~~~~~

node_modules/@wdio/globals/types.d.ts:5:53 - error TS2694: Namespace '"/Users/kyrylo/Projects/webdriverio-types-issue-example/node_modules/webdriverio/build/index"' has no exported member 'MultiRemoteElement'.

5 type MultiRemoteElementType = import('webdriverio').MultiRemoteElement
                                                      ~~~~~~~~~~~~~~~~~~

node_modules/puppeteer-core/lib/esm/puppeteer/common/TargetManager.d.ts:23:51 - error TS2694: Namespace '"/Users/kyrylo/Projects/webdriverio-types-issue-example/node_modules/puppeteer-core/node_modules/devtools-protocol/types/protocol"' has no exported member 'Target'.

23 export type TargetFactory = (targetInfo: Protocol.Target.TargetInfo, session?: CDPSession) => Target;
                                                     ~~~~~~


Found 5 errors in 2 files.

Errors  Files
     4  node_modules/@wdio/globals/types.d.ts:1
     1  node_modules/puppeteer-core/lib/esm/puppeteer/common/TargetManager.d.ts:23
```

You can ignore the error in `puppeteer-core`, I've already fixed it there, should be released with the next version. What I want to solve is the errors in `@wdio/globals/types`

--- a/packages/webdriverio/src/types.ts
+++ b/packages/webdriverio/src/types.ts
@@ -233,11 +233,11 @@ interface InstanceBase extends EventEmitter, SessionFlags {
 export interface BrowserBase extends InstanceBase, CustomInstanceCommands<Browser> {
     isMultiremote: false
 }
+export interface Browser extends BrowserBase, BrowserCommandsType, ProtocolCommands {}
 
 /**
  * export a browser interface that can be used for typing plugins
  */
-interface Browser extends BrowserBase, BrowserCommandsType, ProtocolCommands {}
 export interface ElementBase extends InstanceBase, ElementReference, CustomInstanceCommands<Element> {
     isMultiremote: false
     /**
@@ -272,8 +272,8 @@ export interface ElementBase extends InstanceBase, ElementReference, CustomInsta
      */
     error?: Error
 }
+export interface Element extends ElementBase, ProtocolCommands, Omit<BrowserCommandsType, keyof ElementCommandsType>, ElementCommandsType {}
 
-interface Element extends ElementBase, ProtocolCommands, Omit<BrowserCommandsType, keyof ElementCommandsType>, ElementCommandsType {}
 interface MultiRemoteBase extends Omit<InstanceBase, 'sessionId'>, CustomInstanceCommands<WebdriverIO.MultiRemoteBrowser> {
     /**
      * multiremote browser instance names
@@ -310,7 +310,9 @@ interface MultiRemoteElementBase {
 }
 
 interface MultiRemoteBrowserType extends MultiRemoteBase, MultiRemoteBrowserCommandsType, MultiRemoteProtocolCommandsType { }
+export interface MultiRemoteBrowser extends MultiRemoteBrowserType {}
 interface MultiRemoteElementType extends MultiRemoteElementBase, MultiRemoteProtocolCommandsType, Omit<MultiRemoteBrowserCommandsType, keyof MultiRemoteElementCommandsType>, MultiRemoteElementCommandsType {}
+export interface MultiRemoteElement extends MultiRemoteElementType {}
 
 export type ElementFunction = ((elem: HTMLElement) => HTMLElement) | ((elem: HTMLElement) => HTMLElement[])
 export type CustomStrategyFunction = (...args: any) => ElementReference | ElementReference[]

